force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -299,16 +386,16 @@ public with sharing class FlowControlLoopsCollections {
      * @return The Map of fruits and their quantities.
      */
     public static Map<String, Integer> addItemsToMap() {
-        // Initialize an empty Map
+        Map<String, Integer>fruitsMap = new Map<String, Integer>();// Initialize an empty Map
 
         // Add key-value pairs to the Map
-        // add Apples => 5
-        // add Oranges => 10
-        // add Bananas => 15
-        // add Pears => 20
-        // add Grapes => 25
+        fruitsMap.put('Apples', 5);// add Apples => 5
+        fruitsMap.put('Oranges', 10);// add Oranges => 10
+        fruitsMap.put('Bananas', 15);// add Bananas => 15
+        fruitsMap.put('Pears', 20);// add Pears => 20
+        fruitsMap.put('Grapes', 25);// add Grapes => 25
```

Comment:
Try to add the values of the map in the constructor Try using a static variable for employeeSalaries